src/Styleguide/Components/Lightbox/CloseButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { color, space } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+
+const CloseButtonContainer = styled.svg`
+  width: ${space(6)}px;
+  height: ${space(6)}px;
+  padding: ${space(3) / 2}px;
+  stroke: ${color("white100")};
+  stroke-width: 3px;
+  overflow: visible;
+  cursor: pointer;
+`
+
+export interface CloseButtonProps {
+  onClick: (Event) => void
```

Comment:
I wonder if this is actually typing the parameter as an `Event` or if it’s calling the parameter `Event` with (presumably) an `any` type.